modules/rules_license/0.0.8/presubmit.yml

```diff
@@ -0,0 +1,79 @@
+
+default_tests: &default_tests
```

Comment:
We shouldn't use the full presubmit.yml of rules_license in BCR, 0.0.7 used a very minimal one: https://github.com/bazelbuild/bazel-central-registry/blob/main/modules/rules_license/0.0.7/presubmit.yml

Reply:
@meteorcloudy bazel-central-registry CI validates that the pre-submit matches what's checked in to the actual git repo. So if we want to roll back to the previous tests then we'd need to change that in rules_license and cut a new release.